ksqldb-functional-tests/src/test/resources/query-validation-tests/bytes.json

```diff
@@ -27,6 +27,31 @@
         {"topic": "TEST2", "value": {"B": "dmFyaWF0aW9ucw=="}}
       ]
     },
+    {
```

Comment:
This AVRO test works if you also add the Avro schema translator to RecordNode:

```
    {
      "name": "AVRO in/out",
      "statements": [
        "CREATE STREAM TEST (ID STRING KEY, b BYTES) WITH (kafka_topic='test', value_format='AVRO');",
        "CREATE STREAM TEST2 AS SELECT * FROM TEST;"
      ],
      "topics": [
        {
          "name" : "test",
          "valueSchema" : {
            "type" : "record",
            "name" : "KsqlDataSourceSchema",
            "namespace" : "io.confluent.ksql.avro_schemas",
            "fields" : [ {
              "name" : "ID",
              "type" : [ "null", "string" ],
              "default" : null
            }, {
              "name" : "B",
              "type" : [ "null", "bytes" ],
              "default" : null
            }],
            "connect.name" : "io.confluent.ksql.avro_schemas.KsqlDataSourceSchema"
          },
          "valueFormat" : "AVRO"
        }
      ],
      "inputs": [
        {"topic": "test", "value": {"B": "dmFyaWF0aW9ucw=="}}
      ],
      "outputs": [
        {"topic": "TEST2", "value": {"B": "dmFyaWF0aW9ucw=="}}
      ]
    },
```

Reply:
Thanks. I'll put this in another PR. When added the AVRO translator with an AVRO schema, the `bytes.json` worked, but I got other issues with the `avro.json` 'cause I required that the translator uses WRAP_SINGLES, which later caused other issues in other cases. So I ended up just submitting the code for Protobuf.